package/skelcd-control-MicroOS.changes

```diff
@@ -1,3 +1,8 @@
+-------------------------------------------------------------------
+Mon Jun 28 12:39:32 UTC 2021 - Alberto Planas Dominguez <aplanas@suse.com>
+
+- Add remote attestation roles: micro_os_role_ra_{agent,verifier}
+
```

Comment:
convention is to add version below, so it is easier to see which version cause which change.

```suggestion
- 20210628

```

Reply:
Done.